src/site/includes/metatags.drupal.liquid

```diff
@@ -1,59 +1,66 @@
 <link rel="canonical" href="{{ hostUrl }}{{ entityUrl.path }}" />
+
 {% if changed %}
-<meta name="DC.Date" content="{{changed| dateFromUnix: 'YYYY-MM-DD'}}" property="http://purl.org/dc/terms/date">
+<meta name="DC.Date" scheme="W3CDTF" content="{{ changed| dateFromUnix: 'YYYY-MM-DD' }}">
+{% endif %}
+
+{% if entityUrl.path %}
+<meta content="{{ hostUrl }}/{{ entityUrl.path }}" property="og:url">
+{% else %}
+<meta content="{{ hostUrl }}" property="og:url">
 {% endif %}
+
+<meta content="website" property="og:type">
+
 {% assign tagsSize = entityMetatags | size %}
 {% if tagsSize > 0 %}
-  <!-- Metatags -->
-  {% for tag in entityMetatags %}
-    {% if tag.__typename == "MetaValue" %}
+<!-- Metatags from Drupal -->
+    {% for tag in entityMetatags %}
```

Comment:
It looks like this is adding spaces? Most of our templates are indented with two spaces, rather than four, I think.